src/message.rs

```diff
@@ -1,3 +1,14 @@
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref HOSTNAME: String = {
+        match std::env::var("HOSTNAME") {
+            Ok(value) => value,
+            Err(_) => "undefined".to_string(),
```

Comment:
Maybe "hostname" is better than undefined?

Reply:
When I used undefined, I've thought about that case - we are trying to get a value from an environment variable, but getting errors. It means that happened something wrong with setting this variable on a container or whole server  - it hasn't been set up or set up, but incorrect, and so on.